src/components/Api.js

```diff
@@ -0,0 +1,88 @@
+export default class Api {
+  constructor(options) {
+    this.baseUrl = options.baseUrl;
+    this.header = options.headers;
+  }
+
+  _checkResponse(res) {
```

Comment:
It's great to have a method for checking the server rresponse, so the logic is not duplicated